commit f739780872fe1200993a2509c50803338884e1af
Author: Athanasios Oikonomou <athoik@gmail.com>
Date:   Sat Jan 8 21:23:06 2022 +0200

    Use python3 compatible syntax
    
    Allow building with newer python3 images.

diff --git a/plugin/plugin.py b/plugin/plugin.py
index 741311b..d3b3a6d 100644
--- a/plugin/plugin.py
+++ b/plugin/plugin.py
@@ -22,7 +22,7 @@ from . import _
 from Plugins.Plugin import PluginDescriptor
 
 def main(session,**kwargs):
-	import ui
+	from . import ui
 	session.open(ui.HdmiTest)
 
 def Plugins(**kwargs):
diff --git a/plugin/ui.py b/plugin/ui.py
index 8c44aa2..61c7102 100644
--- a/plugin/ui.py
+++ b/plugin/ui.py
@@ -528,7 +528,7 @@ class HdmiTestInfoScreen(Screen):
 		data = 16 * '\x00'
 		length = message.getData(data, len(data))
 		opcode = ""
-		if opCode.has_key(cmd):
+		if cmd in opCode:
 			if cmd == 0 and length == 0:
 				opcode = opCode[cmd]
 			elif cmd == 0:
diff --git a/setup_translate.py b/setup_translate.py
index 29e8221..7f8e807 100644
--- a/setup_translate.py
+++ b/setup_translate.py
@@ -21,9 +21,9 @@ class build_trans(cmd.Command):
 					if f.endswith('.po'):
 						src = os.path.join(lc, f)
 						dest = os.path.join(lc, f[:-2] + 'mo')
-						print "Language compile %s -> %s" % (src, dest)
+						print("Language compile %s -> %s" % (src, dest))
 						if os.system("msgfmt '%s' -o '%s'" % (src, dest)) != 0:
-							raise Exception, "Failed to compile: " + src
+							raise Exception("Failed to compile: " + src)
 
 class build(_build):
 	sub_commands = _build.sub_commands + [('build_trans', None)]
